lib/utils/inputValidator.js

```diff
@@ -59,6 +61,19 @@ var isNinesTrytes = function(trytes) {
     return isString(trytes) && /^[9]+$/.test(trytes);
 }
 
+/**
+ *  convert the input string to trytes, then convert the resulting trytes to a string,
+ *  and check if the string matches the original input
+ *
+ * @method isSafeString
+ * @param  {string} input string to be checked
+ * @return {boolean}
+ */
+var isSafeString = function(input) {
+
+    return ascii.fromTrytes(ascii.toTrytes(input)) === input;
```

Comment:
I'd suggest to add a check for extended ascii due to compatibility issues (see: https://github.com/iotaledger/iota.lib.js/issues/243).
That should look like:
```js
return /^[\x00-\x7F]*$/.test(input) && ascii.fromTrytes(ascii.toTrytes(input)) === input
```